keyboards/atreus/keymaps/ibnuda/keymap.c

```diff
@@ -0,0 +1,281 @@
+#include QMK_KEYBOARD_H
+
+typedef enum {
+    SINGLE_TAP,
+    SINGLE_HOLD,
+    DOUBLE_TAP,
+} td_state_t;
+
+static td_state_t td_state;
+
+int current_dance(qk_tap_dance_state_t *state);
+
+void dance_tmb_finished(qk_tap_dance_state_t *state, void *user_data);
+void dance_tmb_reset(qk_tap_dance_state_t *state, void *user_data);
+
+// enum for tap dances.
+enum {
+    TD_DLT_CTLDLT = 0,
+    TD_SCLN_CLN,
+    TD_LEFT_THUMB,
+};
+
+// enum for combos.
+enum combos {
+    // left hand combinations.
+    COLON_COMMA,
+    COMMA_DOT,
+    DOT_P,
+    Q_J,
+    J_K,
+
+    // right hand combinations.
+    L_R,
+    R_C,
+    C_G,
+    V_W,
+    W_M,
+
+    // both hands combinations.
+    DOT_C,
+    J_W,
+    P_G,
+    U_H,
+    K_M,
+};
+
+#define _BASE    0
+#define _LOWER   1
+#define _RAISE   2
+#define _ADJUST  3
+#define _MUIS    4
+
+#define ____ KC_TRNS
```

Comment:
QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/1a79f14e4390d088fcf6b26fe3b62549875c3e71/quantum/quantum_keycodes.h#L29-L31

Please delete Line 53, and replace all instances of `____` (four underscores) with `_______` (seven underscores).